wg-captive-browser.sh

```diff
@@ -36,6 +36,7 @@ main() {
   add_veth
   add_iptable_rules
 
+  sudo -u $browser_user -i $browser_command_profile
```

Comment:
This isn't executing in the network namespace so it seems like it defeats the whole purpose of this tool

Reply:
Correct, it also doesn't start a browser session. Firefox needs a profile before it can run. If you don't create the profile beforehand, it will pop up with the Profile Manager. This command circumvents that by creating a profile on disk first.

No network needed for that :-)